pkgs/by-name/ut/ut1999/package.nix

```diff
@@ -41,41 +63,50 @@ in stdenv.mkDerivation rec {
     stdenv.cc.cc
   ];
 
-  nativeBuildInputs = [
+  nativeBuildInputs = lib.optionals (!stdenv.isDarwin) [
     copyDesktopItems
     autoPatchelfHook
     imagemagick
+  ] ++ lib.optionals (stdenv.isDarwin) [
+    undmg
   ];
 
-  installPhase = ''
+  installPhase = let
+    outPrefix = if (!stdenv.isDarwin) then "$out" else "$out/UnrealTournament.app/Contents/MacOS";
```

Comment:
I'd again flip this; I find that easier to read; if it's darwin, then special else normal.